.travis.yml

```diff
@@ -2,11 +2,9 @@ language: node_js
 sudo: false
 install: true
 jdk: oraclejdk11
-node_js: "10.16.0"
+node_js: "8"
 dist: trusty
 
-before_install: if [[ `yarn -v` != 1.17* ]]; then curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.17.3 && export PATH="$HOME/.yarn/bin:$PATH"; fi
```

Comment:
I don't know what the default version is on Travis, let's use `1.15.2` to align with `sonar-enterprise`.
Or update `build.gradle` from `sonar-enterprise` to use a more recent one.